File: resources/Arcadia/Client/EngineService.js
API.onKeyDown.connect(function (sender, key) {
    var player = API.getLocalPlayer();
    if (API.isPlayerInAnyVehicle(player) && API.getPlayerVehicleSeat(player) == -1) {
        if (key.KeyCode === Keys.Up) {
            API.triggerServerEvent("engine_on_off");
        }
    }
});
API.onKeyDown.connect(function (sender, key) {
    var player = API.getLocalPlayer();
    if (API.isPlayerInAnyVehicle(player) && API.getPlayerVehicleSeat(player) == -1) {
        if (key.KeyCode === Keys.M) {
            API.triggerServerEvent("OpenmenuCar");
        }
    }
});

File: resources/Arcadia/Server/Services/VehicleService/EngineService.cs
using CherryMPServer;
//
//
using CherryMPServer.Constant;
using CherryMPServer;
using CherryMPShared;
//;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.IO;

public class EngineService : Script
{
    public EngineService()
    {
        API.onClientEventTrigger += onClientEvent;
    }

    private void onClientEvent(Client sender, string eventName, params object[] arguments)
    {
        if (eventName == "engine_on_off")
        {
            if (sender.vehicle.engineStatus == true)
            {
                sender.vehicle.engineStatus = false;
                API.sendNotificationToPlayer(sender, "Двигатель заглушен");
            }
            else if (sender.vehicle.engineStatus == false)
            {
                sender.vehicle.engineStatus = true;
                API.sendNotificationToPlayer(sender, "Двигатель заведён");
            }
        }
    }

    [Command("engineoff")]
    public void engineOff(Client player)
    {
        if (player.isInVehicle & API.getPlayerVehicleSeat(player) == -1)
        {
            player.vehicle.engineStatus = false;
        }
        else
        {
            API.sendNotificationToPlayer(player, "Вы должны быть на водительском месте.");
        }
    }

    [Command("engineon")]
    public void engineOn(Client player)
    {
        if (player.isInVehicle)
        {
            player.vehicle.engineStatus = true;
        }
        else
        {
            if (player.isInVehicle & API.getPlayerVehicleSeat(player) == -1)
                API.sendNotificationToPlayer(player, "Вы должны быть на водительском месте.");
        }
    }

}